README.md

```diff
@@ -1,2 +1,127 @@
-# clever-csharp
-C# client library for Clever APIs
+# Clever - the C# library for the Clever API
+
+## Installation and API Documentation
+View more detailed documentation [here](docs/README.md)
+
+<a name="getting-started"></a>
+## Getting Started
+
+```csharp
+using System;
+using System.Diagnostics;
+using Clever.Api;
+using Clever.Client;
+using Clever.Model;
+
+namespace Test
+{
+    public static class Program
+    {
+        public static void Main()
+        {
+
+            // Configure OAuth2 access token for authorization: oauth
+            Configuration.Default.AccessToken = "DEMO_TOKEN";
```

Comment:
Might we instead use something like `GetEnvironmentVariable` here, to remind people not to include/commit secrets in their code? 

https://msdn.microsoft.com/en-us/library/77zkk0b6(v=vs.110).aspx

Same idea for `DEMO_TOKEN` on line 65.